source/reference/mongosync.txt

```diff
@@ -91,30 +94,38 @@ Global Options
    Sets the port used by the HTTP server for the {+c2c-product-name+}
    HTTP API.
 
-.. option:: --id <ID>
+.. option:: --reversible
```

Comment:
This is not a command line option we support. Instead, `reversible` is an option for the `/start` API.